packages/web/integrations/notifi/notifi-subscription-card/fetched-card/alert-list.tsx

```diff
@@ -0,0 +1,47 @@
+import { FunctionComponent } from "react";
+
+import { useNotifiConfig } from "../../notifi-config-context";
+import { AlertRow } from "./alert-row";
+
+interface Props {
+  disabled: boolean;
+  toggleStates: Record<string, boolean>;
+  setToggleStates: React.Dispatch<
+    React.SetStateAction<Record<string, boolean>>
+  >;
+}
+
+export const AlertList: FunctionComponent<Props> = (props) => {
+  const config = useNotifiConfig();
+
+  if (config.state !== "fetched") {
+    return null;
+  }
+
+  return (
+    <ul className="mt-[-18px] block px-[40px] pb-[18px] text-[12px] font-[500]">
+      {config.data.eventTypes.map((row) => {
+        if (row.type === "label") {
+          return (
+            <li
+              key={row.name}
+              className="mb-[24px] mt-[36px] flex flex-col gap-[8px]"
```

Comment:
I see absolute pixels used everywhere, is it possible to use REM anywhere? That's our convention in our app.

Reply:
Updated with fix.